autoload/ensime.vim.py

```diff
@@ -9,10 +9,18 @@
 
 def ensime_init_path():
     path = os.path.abspath(inspect.getfile(inspect.currentframe()))
-    if path.endswith('/rplugin/python/ensime.py'):  # nvim rplugin
```

Comment:
I think that wrapping the expected path with [`normpath`](https://docs.python.org/2/library/os.path.html#os.path.normpath) should be enough:

```python
if path.endswith(os.path.normpath('rplugin/python/ensime.py')):
```

It will leave it the same for linux and mac but will replace `/` with `\` when on windows